kata-deploy/scripts/remove-kata-containerd.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+echo "delete kata artifacts"
+rm -rf /opt/kata
+rm -rf /usr/share/defaults/kata-containers
+rm -f /etc/containerd/config.toml
```

Comment:
this is a system administrator file, we should only remove the default configuration.

Reply:
ack, if it existed before, i'll replace it.